### ISSUE TITLE
Additional factory method

### ISSUE THREAD

**Issue by Thomasvdam** [FranklinWaller/ImageCutter.js#1]
Rather than having to wait on the init() call after constructing a new instance, it would be nice to have the option to use an async factory method.

Example:
```JavaScript
const cutter = await createNewCutter('image_url');

cutter.cut(10, 10);
```

**Comment by FranklinWaller** [comment]
Will look into it, ofcourse pull requests are welcome ;)
